game/src/ltn/route.rs

```diff
@@ -44,12 +44,23 @@ impl RoutePlanner {
     }
 
     fn update(&mut self, ctx: &mut EventCtx, app: &App) {
-        self.panel = Panel::new_builder(Widget::col(vec![
+        let new = Panel::new_builder(Widget::col(vec![
             ctx.style()
                 .btn_outline
                 .text("Back to editing modal filters")
                 .hotkey(Key::Escape)
                 .build_def(ctx),
+            Widget::row(vec![
+                Line("Main Road Penalty")
+                    .into_widget(ctx),
+                Spinner::f64_widget(
+                    ctx,
+                    "main road penalty",
+                    (0.5, 10.0),
```

Comment:
Minimum of `1.0` otherwise we'll prefer main roads more than usual. Since this is a multiplier on base time to cross a road, it'd mean somebody is going twice the speed limit O_O